src/components/NavBar/NavbarDesktop.jsx

```diff
@@ -62,12 +62,7 @@ const NavbarDesktop = ({ setShowConfirmation }) => {
       <AppBar position="static" color="primary">
         <Toolbar sx={{ minHeight: '64px' }}>
           <Link to="/" aria-label="Home">
-            <img
-              src="/pass-logo.png"
-              alt="logo"
-              style={{ marginRight: '2rem' }}
-              aria-label="logo"
-            />
+            <img src="/pass-logo.png" alt="CODE PDX logo" style={{ marginRight: '2rem' }} />
```

Comment:
The png in question is the PASS specific logo. The Code PDX has a different look.

Reply:
Ah perfect, will fix 👍